testsuite/coiio/coiiotest.cpp

```diff
@@ -0,0 +1,147 @@
+#include <coiio/imageio.h>
```

Comment:
I'd propose that maybe this would be a better test if it were `coiiotest.c` -- C -- so we don't make any mistakes that would make it not work with a true C compiler.